src/config.rs

```diff
@@ -1935,6 +1935,9 @@ pub struct TiKvConfig {
     #[config(skip)]
     pub log_file: String,
 
+    #[config(skip)]
+    pub log_format: LogFormat,
```

Comment:
You need to update "tests/integrations/config/mod.rs" to test if it can be serialized successfully.

Reply:
Added at b70aba8